spaceship/static/wip-overlay.css

```diff
@@ -0,0 +1,78 @@
+#overlay-background {
```

Comment:
Personally I avoid id selectors, even when there's only one, cause I've lost way too many specificity wars... :) it's a little obscure but I thought you might be interested because you mentioned css. 
https://dev.to/claireparkerjones/reasons-not-to-use-ids-in-css-4ni4

Reply:
Ahhhh...I have a feeling this can explain some crazy CSS behaviors I've pulled my hair out about before.

Reply:
![css-specificity-wars](https://user-images.githubusercontent.com/395397/57121565-45972500-6d2d-11e9-89bd-e222c6c76983.png)